Issue and PR templates

danez opened the pull request:
This is the starting point for Issue and pr templates.

Maybe also a table in the PR template for the env information?

hzoo commented:
We should make it correspond to the labels (which we can also change https://github.com/babel/babel/blob/master/lerna.json#L6-L14) https://github.com/babel/babel/labels

hzoo commented on .github/ISSUE_TEMPLATE.md:
```diff
@@ -0,0 +1,53 @@
+If you have a support request or question please submit them to on of this resources:
+
+* Discussion Forum: https://discuss.babeljs.io/
+* StackOverflow: http://stackoverflow.com/questions/tagged/babeljsusing the tag `[babeljs]`
```
needs a space here right?

hzoo commented on .github/ISSUE_TEMPLATE.md:
```diff
@@ -0,0 +1,53 @@
+* StackOverflow: http://stackoverflow.com/questions/tagged/babeljsusing the tag `[babeljs]`
+* Slack community: https://slack.babeljs.io/
+
+Issues on github are only issues related to development of babelitself.
```
here too

hzoo commented:
Is it worth mentioning contributing or linking the readme?

hzoo commented:
You can just make it commit if you want, and use `[skip ci]` if you want too

danez commented:
I changed the PR template to link to the Contributing.md and the issue template to link to the readme.md. 
Fixed all the typos, spaces, etc. and made the version info a table.

tarlepp commented:
+1 for this.

hzoo commented:
Aside: just 👍 with reactions please.

How would we go about asking for the versions of the packages? Since `Babel` version is really the babel-core version.

It might be a lot of info but I think we also care about the versions of any relevant babel-* pkgs

hzoo commented:
Oh another suggestion: instead of just input/output/config - even making a reproducible repo or the simplest is to just use the REPL and post a link to it. When we are able to import any plugin into the repl that would be even better (https://github.com/babel/babel.github.io/issues/858)

danez commented:
About the babel version: is there a command with npm maybe to ask for all versions of all installed babel-packages? like `npm ls babel*` that would be nice.
